docs/src/user-guide/command-line-interface.md

```diff
@@ -8,6 +8,10 @@ eleventyNavigation:
 
 ---
 
+The ESLint CLI is a command line interface that lets you execute linting from the terminal. The CLI has a variety of options that you can pass to its commands.
```

Comment:
```suggestion
The ESLint CLI lets you execute linting from the terminal. The CLI has a variety of options that you can pass to its commands.
```
we already have command line interface in the name itself? what do you think?

Reply:
changing to:

```suggestion
The ESLint Command Line Interface (CLI) lets you execute linting from the terminal. The CLI has a variety of options that you can pass to its commands.
```